Vagrantfile

```diff
@@ -6,7 +6,7 @@ Vagrant.configure(2) do |config|
   config.vm.define "qmk_firmware"
 
   # VMware/Virtualbox ( and also Hyperv/Parallels) 64 bit
-  config.vm.box = "bento/ubuntu-16.04"
+  config.vm.box = "generic/debian9"
```

Comment:
Is there a `generic/debian`, i.e. latest, with no version number? That way the Dockerfile (which uses `FROM debian`) and the Vagrantfile will keep themselves up to date, and we won't need to update this once Debian 10 is out (which will be in a couple of months).

Reply:
The official debian boxes use the code name, [like debian/stretch64](https://app.vagrantup.com/debian/boxes/stretch64) and they only build for virtualbox, not VMware which has been in the Vagrantfile for some time. We could consider locking the Docker image to `debian:9` if that helps keep things consistent?

Reply:
Yes, that would help.

Reply:
Done, I've also noted the image size is rather large but i will tackle that in a separate PR.